README.adoc

```diff
@@ -1,68 +1,205 @@
 :uri-mozilla-addon: https://addons.mozilla.org/firefox/addon/asciidoctorjs-live-preview
 :uri-opera-addon: https://addons.opera.com/extensions/details/asciidoctorjs-live-preview
 :uri-edge-webstore: https://microsoftedge.microsoft.com/addons/detail/asciidoctorjs-live-previ/pefkelkanablhjdekgdahplkccnbdggd
+// Webstore graphics
+:uri-chrome-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png
+:uri-mozilla-addon-image: https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg
+:uri-opera-addon-image: https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png
+:uri-edge-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png
 
-This project uses https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] to preview AsciiDoc as HTML inside Chrome (or Chromium), Firefox and Opera!
+Enjoy AsciiDoc everywhere your browser goes with the {project-name} (the Extension).
 
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png[link={uri-chrome-webstore},height="64px",width="212px"]
-image:https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg[link={uri-mozilla-addon},height="64px",width="184px"]
-image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png[link={uri-opera-addon},height="64px",width="227px"]
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png[link={uri-edge-webstore},height="64px",width="177px"]
+The {project-name} uses https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] to preview AsciiDoc files (.ad, .adoc, .asc, .asciidoc, and optionally .txt) as HTML inside Chrome (or Chromium), Brave, Firefox, Opera and Microsoft Edge.
 
 *Project health:* image:https://github.com/asciidoctor/asciidoctor-browser-extension/workflows/CI/badge.svg[Build Status, link="https://github.com/asciidoctor/asciidoctor-browser-extension/actions?query=workflow%3ACI"]
 
-== Usage
+== Get started
+
+To get started, you will do three things in total:
+
+. Install the {project-name} extension
+. Configure the browser's options - What the browser allows the {project-name} to do
+. Review and configure your {project-name} specific options
+
+=== Browser webstore install
+Use the webstore links on the images below for direct access to the plugin.
+You may also search your browser's webstore for the Asciidoctor Live Preview extension.
+
+[%autowidth,cols="a,a,a,a",frame=none,grid=none,stripes=none]
+|===
+|
+.Chrome, Chromium, or Brave
+image:{uri-chrome-webstore-image}[link={uri-chrome-webstore},height="64px",width="212px"]
+
+|
+.FireFox
```

Comment:
```suggestion
.Firefox
```

(nitpick): I believe that the correct capitalization is "Firefox"

Reply:
Fixed it in the other places too. 👍